data/completion/etelpmoc.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+# etelpmoc is the backwards half of complete: it de-serialises the tab
+# completion request into the appropriate environs expected by the tab
+# completion tools, performs whatever action is wanted, and serialises the
+# result. It accomplishes this by having functions override the builtin
+# completion commands.
+#
+# this always runs "inside", in the same environment you get when doing "snap
+# run --shell", and snap-exec is the one setting the first argument to the
```

Comment:
@chipaca - no. I was thinking you had work and then in retrospect realized it needs to be the way it is.